ci/test-abi/google_cloud_cpp_common.expected.abi.dump

```diff
@@ -0,0 +1,564 @@
+$VAR1 = {
+          'ABI_DUMPER_VERSION' => '1.1',
+          'ABI_DUMP_VERSION' => '3.5',
+          'Arch' => 'x86_64',
+          'Compiler' => 'GNU C++11 7.3.0 -mtune=generic -march=x86-64 -g -std=gnu++11 -fPIC -fstack-protector-strong',
+          'Headers' => {
+                         'build_info.h' => 1,
+                         'setenv.h' => 1,
+                         'throw_delegate.h' => 1
+                       },
+          'Language' => 'C++',
+          'LibraryName' => 'libgoogle_cloud_cpp_common.so',
+          'LibraryVersion' => '0.1.0',
+          'MissedOffsets' => '1',
+          'MissedRegs' => '1',
+          'NameSpaces' => {
+                            '__gnu_cxx' => 1,
+                            'google::cloud::v0::internal' => 1,
+                            'std' => 1,
+                            'std::__cxx11' => 1,
+                            'std::__exception_ptr' => 1
+                          },
+          'Needed' => {
+                        'libc.so.6' => 1,
+                        'libgcc_s.so.1' => 1,
+                        'libstdc++.so.6' => 1
+                      },
+          'PublicABI' => '1',
+          'Sources' => {
+                         'build_info.cc' => 1,
+                         'setenv.cc' => 1,
+                         'throw_delegate.cc' => 1
+                       },
+          'SymbolInfo' => {
+                            '16246' => {
```

Comment:
Do these numbers really stay static across builds, compilers, etc.? Or are they internal numbers, and it is just a marker, but is compatible as long as there's correspondence between the id matching for function input / output args...?

Reply:
These numbers also can change from build to build, but the compatibility reports are not affected.